.github/workflows/PR_Test_CI.yml

```diff
@@ -0,0 +1,62 @@
+# This workflow will install Python dependencies, run tests and lint with a single version of Python
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-python-with-github-actions
+
+name: PR Test CI
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
Why Ubuntu?  Why not CentOS or Fedora?

Reply:
Its the recommend light and agile machine for Linux

Reply:
Given that the real thing will be running on rhcos, I strongly urge you to consider using Fedora or CentOS as the base.  Any of these images will boot quickly enough.

Reply:
Those are the options: https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on

Reply:
OK, fair enough then.